portfolio/src/main/webapp/index.html

```diff
@@ -240,6 +204,90 @@ <h1>Projects</h1>
 
         </div>
 
+        <div class="hidden" id="chart-wrapper">
+            <div class="" id="chart-container"></div>
+            <button onclick="toggleChart(true)">Hide Chart</button>
+        </div>
+        <div class="part" id="chartpart">
+            <div id="voting-stats">
+                <h1 class="chartpart-header">What's your favorite music genre?</h1>
+                <div class="buttonCenter">
+                    <button class="voting-buttons" 
+                            onclick="toggleChart(false)" 
+                            id="chartViewer">View Pie Chart</button>
+                </div>
+                <table id="voting-stats-table">
+                    <tr class="voting-table-nonNums">
+                        <td>Hip-Hop/Rap</td>
+                        <td>Country</td>
+                        <td>Pop</td>
+                        <td>Rock</td>
+                        <td>Classical</td>
+                        <td>R&amp;B</td>
+                    </tr>
+                    <tr class="voting-table-nums">
+                        <td id="HipHop">0</td>
```

Comment:
nit: id's should be lowercase. For multi-word id's, separate with dashes (hip-hop and r-and-b)

Reply:
Noted